uploaders/xcarchiveuploader.go

```diff
@@ -0,0 +1,118 @@
+package uploaders
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/bitrise-io/go-utils/log"
+	"github.com/bitrise-io/go-xcode/exportoptions"
+	"github.com/bitrise-io/go-xcode/ipa"
+	"github.com/bitrise-io/go-xcode/plistutil"
+	"github.com/bitrise-io/go-xcode/profileutil"
+	xcarchive "github.com/bitrise-io/go-xcode/xcarchive"
+)
+
+// DeployXcarchive ...
+func DeployXcarchive(pth, buildURL, token, notifyUserGroups, notifyEmails, isEnablePublicPage string) (string, error) {
+	log.Printf("analyzing xcarchieve")
+
+	infoPlistPth, err := xcarchive.UnwrapEmbeddedInfoPlist(pth)
+	if err != nil {
+		return "", fmt.Errorf("failed to unwrap Info.plist from xcarchive, error: %s", err)
+	}
+
+	infoPlistData, err := plistutil.NewPlistDataFromFile(infoPlistPth)
+	if err != nil {
+		return "", fmt.Errorf("failed to parse Info.plist, error: %s", err)
+	}
+
+	appTitle, _ := infoPlistData.GetString("CFBundleName")
```

Comment:
I think most of these infos are not valid in case of an xcarchive

Reply:
As I see, athives have the same  provsion profile as ipa-s.